cfn-resources/test/e2e/utility/util.go

```diff
@@ -98,7 +99,13 @@ func NewClients(t *testing.T) (cfnClient *cfn.Client, atlasClient *admin.APIClie
 func ExecuteGoTemplate(templatePath string, data any) (string, error) {
 	var cfnGoTemplateStr bytes.Buffer
 	name := path.Base(templatePath)
-	cfnGoTemplate, err := template.New(name).ParseFiles(templatePath)
+	cfnGoTemplate := template.New(name).Funcs(map[string]any{
```

Comment:
curious why this is changed

Reply:
Needed by the [go template](https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/4f49b80ec035ddc64d2f28e9c7669f44dfc870d1/cfn-resources/test/e2e/project/project.json.template#L27)

Specific line:
`"Tags": {{ marshal .Tags }}`

I could have treated `.Tags` as a json string in the template instead. But I think it is cleaner to use a template function